Assets/Plugins/CountlySDK/Services/InitializationCountlyService.cs

```diff
@@ -20,6 +21,7 @@ internal InitializationCountlyService(CountlyConfiguration configuration, Countl
         internal async Task OnInitialisationComplete()
         {
             await StartSession();
+           _consentService.SendConsentChanges(_consentService._countlyConsents.Keys.ToList(), true);
```

Comment:
This should not be here. Initial consent state should be sent after the SDK has initialised.

Reply:
OnInitialisationComplete get after initialization of sdk.

Reply:
Mkey.
Consent changes should be sent before starting a session